src/app.controller.ts

```diff
@@ -7,6 +7,10 @@ export class AppController {
 
   @Get()
   getHello(): string {
+    /**
+ * @todo Start Coding
+ * @body Testing TODO BOT
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [asterisks in jsdoc must be aligned](https://app.codacy.com/app/code.diogo.adao/nest-time-tracker/pullRequest?prid=4513240)